config/test.json

```diff
@@ -1,4 +1,5 @@
 {
+  "port": "3061",
```

Comment:
@znarf what do you think about an inconsistent API port for the sake of being able to run tests while also running a dev server?

Reply:
Sounds good I think? I have seen similar attempts at this in other places. Actually for cron jobs search for ```process.env.PORT = 3066;```

Reply:
I'm not really sure what having a consistent API port for this gives us. Seems like running tests + running the dev server concurrently should be a priority for a productive dev setup.